<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD>
<META http-equiv=Content-Type content="text/html; charset=us-ascii">
<META content="MSHTML 6.00.2900.3243" name=GENERATOR></HEAD>
<BODY>
<DIV><FONT face=Arial size=2><SPAN class=383395715-24012008>I'm having 
some&nbsp;difficulty geting the Open MPI checkpoint/restart fault tolerance 
working.&nbsp; I have compiled Open MPI with the "--with-ft=cr" flag, but when I 
attempt to run my test program (ring), the ompi-checkpoint command 
fails.&nbsp;&nbsp;I have verified that&nbsp;the test program works fine without 
the fault tolerance enabled.&nbsp; Here are the details:</SPAN></FONT></DIV>
<DIV><FONT face=Arial size=2><SPAN 
class=383395715-24012008></SPAN></FONT>&nbsp;</DIV>
<DIV><FONT face=Arial size=2><SPAN 
class=383395715-24012008>&nbsp;&nbsp;&nbsp;&nbsp; [me@dev1 ~]$ mpirun -np 4 -am 
ft-enable-cr ring<BR><SPAN class=383395715-24012008>&nbsp;&nbsp;&nbsp;&nbsp; 
</SPAN>[me@dev1 ~]$ ps -efa | grep mpirun<BR><SPAN 
class=383395715-24012008>&nbsp;&nbsp;&nbsp;&nbsp; 
</SPAN>me&nbsp;&nbsp;&nbsp;&nbsp; 3052&nbsp; 2820&nbsp; 1 08:25 
pts/2&nbsp;&nbsp;&nbsp; 00:00:00 mpirun -np 4 -am ft-enable-cr 
ring</SPAN></FONT></DIV>
<DIV><FONT face=Arial size=2></FONT>&nbsp;</DIV><FONT><SPAN 
class=383395715-24012008>
<DIV><FONT face=Arial size=2></FONT><BR><FONT face=Arial><FONT size=2><SPAN 
class=383395715-24012008>&nbsp;&nbsp;&nbsp;&nbsp; </SPAN>[me@dev1 ~]$ 
ompi-checkpoint 3052<BR><SPAN class=383395715-24012008>&nbsp;&nbsp;&nbsp;&nbsp; 
</SPAN>[dev1.acme.local:03060] [NO-NAME] ORTE_ERROR_LOG: Unknown error: 5854512 
in file sds_singleton_module.c at line 50<BR><SPAN 
class=383395715-24012008>&nbsp;&nbsp;&nbsp;&nbsp; </SPAN>[dev1.acme.local:03060] 
[NO-NAME] ORTE_ERROR_LOG: Unknown error: 5854512 in file runtime/orte_init.c at 
line 311<BR><SPAN class=383395715-24012008>&nbsp;&nbsp;&nbsp;&nbsp; 
</SPAN>--------------------------------------------------------------------------<BR><SPAN 
class=383395715-24012008>&nbsp;&nbsp;&nbsp;&nbsp; </SPAN>It looks like orte_init 
failed for some reason; your parallel process is<BR><SPAN 
class=383395715-24012008>&nbsp;&nbsp;&nbsp;&nbsp; </SPAN>likely to abort.&nbsp; 
There are many reasons that a parallel process can<BR><SPAN 
class=383395715-24012008>&nbsp;&nbsp;&nbsp;&nbsp; </SPAN>fail during orte_init; 
some of which are due to configuration or<BR><SPAN 
class=383395715-24012008>&nbsp;&nbsp;&nbsp;&nbsp; </SPAN>environment 
problems.&nbsp; This failure appears to be an internal failure;<BR><SPAN 
class=383395715-24012008>&nbsp;&nbsp;&nbsp;&nbsp; </SPAN>here's some additional 
information (which may only be relevant to an<BR><SPAN 
class=383395715-24012008>&nbsp;&nbsp;&nbsp;&nbsp; </SPAN>Open MPI 
developer):</FONT></FONT></DIV>
<DIV><FONT face=Arial size=2></FONT>&nbsp;</DIV>
<DIV><FONT face=Arial><FONT size=2><SPAN 
class=383395715-24012008>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</SPAN>&nbsp; 
orte_sds_base_set_name failed<BR><SPAN 
class=383395715-24012008>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</SPAN>&nbsp; --&gt; 
Returned value Unknown error: 5854512 (5854512) instead of 
ORTE_SUCCESS</FONT></FONT></DIV>
<DIV><FONT face=Arial size=2></FONT>&nbsp;</DIV>
<DIV><FONT face=Arial><FONT size=2><SPAN 
class=383395715-24012008>&nbsp;&nbsp;&nbsp;&nbsp; 
</SPAN>--------------------------------------------------------------------------<BR></FONT></FONT></DIV>
<DIV><SPAN class=383395715-24012008><FONT face=Arial size=2>Any help would be 
appreciated.&nbsp; Thanks.</FONT></SPAN></DIV></SPAN></FONT></BODY></HTML>

